pxtblocks/fields/field_imagedropdown.ts

```diff
@@ -92,17 +184,24 @@ export class FieldImageDropdown extends FieldDropdown implements FieldCustom {
                 // This icon is selected, show it in a different colour
                 backgroundColor = (this.sourceBlock_ as Blockly.BlockSvg).getColourTertiary();
                 button.setAttribute('aria-selected', 'true');
+                this.activeDescendantIndex = i;
+                contentDiv.setAttribute('aria-activedescendant', button.id);
+                button.setAttribute('class', `blocklyDropDownButton ${e ? "blocklyDropDownButtonHover" : "blocklyDropDownButtonFocus"}`);
```

Comment:
This is setting a keyboard focus indicator on the currently selected icon when the dropdown is opened by anything other than a mouse event. Ideally, we'd only do this when we knew it was a keyboard event. The same code is in the field_images.ts file.